template.go

```diff
@@ -7,44 +7,35 @@ import (
 	"github.com/Masterminds/sprig"
 	"io/ioutil"
 	"log"
+	"os"
 	"path/filepath"
 	"text/template"
 )
 
-type OptionalString struct {
-	ptr *string
-}
-
-func (s OptionalString) String() string {
-	if s.ptr == nil {
-		return ""
+func Require(arg string) (string, error) {
+	fmt.Fprintf(os.Stderr, "WARNING: require built-in function will be deprecated in future release. Use required instead.\n")
```

Comment:
I would prefer (is deprecated instead of will be deprecated), e.g.:

DEPRECATION WARNING:  `require` build-in function has been deprecated in favor of \`required\`.`